### ISSUE TITLE
train on pytorch1.5

### ISSUE THREAD

**Issue by xingaoli** [ruotianluo/pytorch-faster-rcnn#171]
It is all right when I only switch the torch version to 1.1. But I find the loss is nan after 20 iters when I try to train the model on torch1.5. I confirm that only the PyTorch version is different. Do you know what this is about?
